x-pack/plugins/enterprise_search/public/applications/shared/setup_guide/cloud/instructions.tsx

```diff
@@ -23,18 +23,18 @@ export const CloudSetupInstructions: React.FC<Props> = ({ productName, cloudDepl
       steps={[
         {
           title: i18n.translate('xpack.enterpriseSearch.setupGuide.cloud.step1.title', {
-            defaultMessage: 'Edit your Elastic Cloud deployment’s configuration',
+            defaultMessage: 'Edit your deployment’s configuration',
           }),
           children: (
             <EuiText>
               <p>
                 <FormattedMessage
                   id="xpack.enterpriseSearch.setupGuide.cloud.step1.instruction1"
-                  defaultMessage="{visitCloudLink} and select your deployment. Within your deployment, select “Edit” from the sidebar to manage your deployment’s configuration."
+                  defaultMessage="Visit the Elastic Cloud console to {editDeploymentLink}."
```

Comment:
Where is `editDeploymentLink` defined? Perhaps `visitCloudLink `?

Reply:
derp, I 100% forgot to update the value 💀 thanks Scotty!!

Reply:
Haha good news though, Kibana's yelling at me about it too

Reply:
https://github.com/elastic/kibana/pull/84584/commits/383dbae1889e0ed576b9981fe8ffd66e9de062fe